dashboard/app/controllers/levels_controller.rb

```diff
@@ -79,13 +79,18 @@ def update_blocks
   # PATCH/PUT /levels/1
   # PATCH/PUT /levels/1.json
   def update
-    if level_params[:name] &&
-        @level.name != level_params[:name] &&
-        @level.name.downcase == level_params[:name].downcase
-      # do not allow case-only changes in the level name because that confuses git on OSX
-      @level.errors.add(:name, 'Cannot change only the capitalization of the level name (it confuses git on OSX)')
-      render json: @level.errors, status: :unprocessable_entity
-      return
+    if level_params[:name] && @level.name != level_params[:name]
+      if @level.name.downcase == level_params[:name].downcase
+        # do not allow case-only changes in the level name because that confuses git on OSX
+        @level.errors.add(:name, 'Cannot change only the capitalization of the level name (it confuses git on OSX)')
+        render json: @level.errors, status: :unprocessable_entity
+        return
+      elsif @level.name == level_params[:name].strip
```

Comment:
hm, should we just always strip the name param? I can't imagine anyone is doing this intentionally